ci/container/container-run.sh

```diff
@@ -27,6 +27,10 @@ Script uses dfinity/ic-build image by default.
 EOF
 }
 
+eprintln() {
+    echo "$@" >&2
+}
+
```

Comment:
This should be moved above the `eprintln` calls above otherwise they would be undefined.